src/languages/handlebars.js

```diff
@@ -225,14 +289,28 @@ export default function(hljs) {
         className: 'template-tag',
         begin: /\{\{\{\{\//,
         end: /\}\}\}\}/,
-        contains: [CLOSING_BLOCK_MUSTACHE_CONTENTS]
+        contains: [ CLOSING_BLOCK_MUSTACHE_CONTENTS ]
+      },
+      // open angle-brocket component invocation
+      {
+        className: 'template-tag',
+        begin: /<:?/,
+        end: />/,
+        contains: [ OPENING_BLOCK_MUSTACHE_CONTENTS, BLOCK_PARAMS, KEYWORD_CONTENTS ]
```

Comment:
You can't put an empty block like KEYWORD_CONTENTS here, it'll do nothing because it has no `match` or `begin` tags.  If you ant to match keywords here you'd put keywords at the top-level.

```suggestion
        className: 'template-tag',
        keywords: // keywords go here
        begin: /<:?/,
        end: />/,
        contains: [ OPENING_BLOCK_MUSTACHE_CONTENTS, BLOCK_PARAMS ]
```